tests/acceptance/features/webUITrashbin/trashbinRestore.feature

```diff
@@ -115,3 +115,40 @@ Feature: Restore deleted files/folders
     And as "user1" file "simple-folder-renamed/file-to-delete-and-restore" should not exist
     #And as "user1" file "simple-folder-renamed/file-to-delete-and-restore" should exist
     And as "user1" file "simple-folder/file-to-delete-and-restore" should not exist
+
+  @issue-B
+  Scenario: Delete and restore file with same name as folder
+    Given the user has created file "a1.txt"
+    And the following files have been deleted by user "user1"
+      | name   |
+      | a1.txt |
+    And the user has created folder "a1.txt"
+    And the following folders have been deleted by user "user1"
+      | name   |
+      | a1.txt |
+    When the user browses to the trashbin page
+    And the user restores file "a1.txt" from the trashbin using the webUI
+    Then the success message "a1.txt was restored successfully" should be displayed on the webUI
+    And file "a1.txt" should not be listed on the webUI
+    And the folder "a1.txt" should be listed on the webUI
+    When the user browses to the files page using the webUI
```

Comment:
check that using the API

Reply:
this is not done yet

Reply:
Couldn't check this using API as both the file and folder has same name and it couldn't separate whether file or folder is present in the trashbin. So the api result is not as expected and the test fails